threeML/plugins/OGIP/eventlist.py

```diff
@@ -461,9 +461,12 @@ def peek(self):
             info_dict['Active Polynomial Counts'] = self._poly_counts.sum()
             info_dict['Unbinned Fit'] = self._unbinned
 
-            S = li_and_ma(self._counts.sum(), self._poly_counts.sum())
+            sig =Significance(self._counts.sum(), self._poly_counts.sum())
 
-            info_dict['Li and Ma Sigma'] = S  # not sure if li and ma applies here
+            bkg_sig = np.sqrt( (self._poly_count_err**2).sum())
+
+
+            info_dict['Li and Ma Sigma'] = sig.li_and_ma_equivalent_for_gaussian_background(bkg_sig)  # not sure if li and ma applies here
```

Comment:
We shouldn't probably call this 'Li and Ma Sigma' since it is not ;-) Maybe "background-corrected significance"?